media/BUILD.gn

```diff
@@ -203,7 +203,10 @@ test("media_unittests") {
   ]
 
   if (is_cobalt) {
-    deps += ["//media/starboard:unit_tests"]
+    # TODO(b/379934658): add starboard_renderer_test.
```

Comment:
Please use the TODO format: TODO(cobalt, b/379934658)

Reply:
Done.